### PR TITLE
Merge hotfix 1.9.1 into develop in preparation for CLI release

### PR DESCRIPTION
**Pull request by wshands**
This branch was created from hotfix/1.9.1 and merging develop into it. Now we will merge this branch back into develop as a surrogate for merging hotfix/1.9.1 into develop

### CODE REVIEW

File: scripts/install-tests.sh [hunk 1]
```diff
@@ -53,6 +53,40 @@ if [ "${TESTING_PROFILE}" = "singularity-tests" ]; then
     singularity --version
 fi
 
+if [ "${TESTING_PROFILE}" = "singularity-tests" ]; then
```
Comment (coverbeck) [inline scripts/install-tests.sh:56]: Why did this file change? I don't see any commits in the hotfix branch that modified it.

Comment (wshands) [reply scripts/install-tests.sh]: Fixed; it looks like I somehow duplicate the block

**Review by Richard-Hansen — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]
When tests pass

**Review by coverbeck — APPROVED** [review]
Pending tests passing

**Pull request merged**